pocs/utils/database.py

```diff
@@ -1,34 +1,50 @@
-import os
-import pymongo
-
+from datetime import date
```

Comment:
Curious why these were moved to the top and `os` and `pymongo` moved down.

Reply:
Putting standard python libraries at the top, then third party, then our own. A style I'm used to... from you know where.
 Probably should discuss whether we should do that globally before making such changes.

Reply:
I was mostly confused by moving the `os` below the `datetime`. Makes sense for `pymongo`. Not a huge deal.